GrammarLinter.md

```diff
@@ -0,0 +1,81 @@
+## English Grammar Linter (Vale)
+
+This repository includes `beta` rules based on the [Vale grammar linter](https://docs.errata.ai/vale). To install Vale on a Mac, run:
+
+```
+brew install vale
+```
+
+You can set up Vale with several different IDEs. For this repository,
+I've tested the integration between Vale and VSCode, and described some tips and tricks
+in [Use Vale in Your IDE](#use-vale-in-your-ide).
+
+Vale pulls rules from YAML files in the `styles/` subdirectory. They include grammar rules in the following subdirectories:
+
+- Modified rules from GitLab in the `styles/cobalt/` subdirectory
+- [Google Developer Style Guide](https://github.com/errata-ai/Google) rules, customized for Vale, in the `styles/Google` subdirectory
+- Rules associated with the [write-good](https://github.com/btford/write-good) grammar linter
+
+These rules are a "Work in Progress"; we may overrule/modify them as we use them to review Cobalt content. For example, if you find a common word / acronym that we use, you're
+welcome to add it (with a PR) to our `styles/cobalt/spelling-exceptions.txt` file.
+
+For more information, see the [Vale documenmtation](https://docs.errata.ai/).
+
+### Vale Configuration
+
+The Vale configuration file is .vale.ini. In this file, we see: 
+
+- The `StylesPath` points to rules in the `styles/` subdirectory.
+- The `BasedOnStyles` parameter specifies style subdirectories.
+- The `IgnoredScopes` tells Vale to ignore content such as code samples, as described in [Vale Documentation](https://docs.errata.ai/vale/config#ignoredscopes).
+
+### Use Vale in Your Repository
+
+If you want to use this Vale configuration in your repository, copy:
+
+- `.vale.ini`
+- The `styles/` subdirectory
+
+If you copy Vale configuration files to your repository, and want to "change the rules",
+open a PR against this cobalt-product-public-docs repo.
+
+### Use Vale in Your IDE
+
+You can set up Vale with several different IDEs. For more information, see the
+[Integrations](https://docs.errata.ai) section of the Vale documentation.
```

Comment:
We can point directly to the integrations section. Same for line 66

```suggestion
[Integrations](https://docs.errata.ai/#integrations) section of the Vale documentation.
```

Alternatively, you could use a MarkDown feature where you can define a link and use it repeatedly in the document, e.g.:

```
<!-- other text -->

[Vale Integrations] section of the Vale documentation.

<!-- other text, until the end of the document -->

[Vale Integrations]: https://docs.errata.ai/#integrations